tendrl/commons/objects/job/__init__.py

```diff
@@ -33,17 +33,6 @@ def save(self, ttl=None):
             _parent.children = list(set(_children))
             _parent.save()
 
-        if self.status == "failed":
-            if "parent" in self.payload:
-                _parent = Job(job_id=self.payload['parent']).load()
-                _msg = "\n Child job %s failed" % self.job_id
-                if _parent.errors:
```

Comment:
this is moved to parent import atom

Reply:
Can you refer the commons change here?

Reply:
https://github.com/Tendrl/commons/pull/978/files#diff-5ef679f35c2d083d057b92f291c0f8e6R208